### PR TITLE
Blog app - controllers specs

### PR DESCRIPTION
**Pull request by Mithi-code**
### Write tests for controllers.

- Add RSpec as a gem to your project.
- Create a Request spec file for all your controllers.
- Check actions If response status was correct.
- Check actions If a correct template was rendered.
- Check actions If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/request/users_spec.rb [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'response body includes correct placeholder text' do
+      expect(response.body).to include('single post')
+    end
+
+    it 'renders a correct template' do
+      expect(response).to render_template(:post)
+    end
```
Comment (Lordkaito) [inline spec/request/users_spec.rb]: - [ ] Please make sure to move this code into a different file so you can have `users_spec` and `posts_spec`.

**Review by Lordkaito — APPROVED** [review]
Hi @Mithi-code, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/request/users_spec.rb (hunk 1 above)
Comment (Mithi-code) [reply spec/request/users_spec.rb]: sure i will do that thanks for your feedback

**Review by BrianSammit — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello @Mithi-code, 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**